From c8c9dc0e3d686096844f1d38b9a02354f593e2b0 Mon Sep 17 00:00:00 2001
From: Minjae Kim <nate.kim@lge.com>
Date: Thu, 21 Mar 2019 19:58:34 +0900
Subject: [PATCH] fix build error by gcc 8.2.0

:Release Notes:

:Detailed Notes:

:Testing Performed:

:QA Notes:

:Issues Addressed:
[GF-9999] Summary
---
 include/llvm/ExecutionEngine/Orc/OrcRemoteTargetClient.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/llvm/ExecutionEngine/Orc/OrcRemoteTargetClient.h b/include/llvm/ExecutionEngine/Orc/OrcRemoteTargetClient.h
index da02250ba16..7105d752a8d 100644
--- a/include/llvm/ExecutionEngine/Orc/OrcRemoteTargetClient.h
+++ b/include/llvm/ExecutionEngine/Orc/OrcRemoteTargetClient.h
@@ -713,7 +713,7 @@ private:
 
   uint32_t getTrampolineSize() const { return RemoteTrampolineSize; }
 
-  Expected<std::vector<char>> readMem(char *Dst, JITTargetAddress Src,
+  Expected<std::vector<uint8_t>> readMem(char *Dst, JITTargetAddress Src,
                                       uint64_t Size) {
     // Check for an 'out-of-band' error, e.g. from an MM destructor.
     if (ExistingError)
-- 
2.17.1

